docs/reference/conditions.md

```diff
@@ -42,6 +42,14 @@ For example, `every' 3 0 (fast 2)` will speed up the cycle on cycles 0,3,6,… w
 
 With this in mind, setting the second argument of `every'` to `0` gives the equivalent `every` function. For example, `every 3` is equivalent to `every' 3 0`.
 
+**Note**: The `every` functions applys to other functions but not directly on the cycle. You can use `every silent` or `every mask ("~")` to silence a full cycle. Mask gives additional flexibility to turn on/off individual steps.
```

Comment:
Typo: applys -> apply (or applies if it had been singular)

Unfortunately I don't understand "The `every` functions applys to other functions but not directly on the cycle."

Reply:
Thanks for the feedback and for checking my spelling (oops) and wording. I'm trying to say that you can't use "every x" to control the cycle directly. But I agree that my sentence isn't clear. 

Here is an alternative - make this a "tip" - which makes it stand out from the "every" functions. Then delete the unclear sentence and have this:

:::tip: The `every` functions can be used to silence a full cycle or part of a cycle by using `silent` or `mask ("~")`. Mask gives additional flexibility to turn on/off individual steps.

d1 $ every 3 silent $ n "2 9 11 2" # s "hh27"
d1 $ every 3 (mask "~") $ n "2 9 10 2" # s "hh27"
d1 $ every 3 (mask "0 0 0 0") $ n "2 9 11 2" # s "hh27" :::

If this is OK, how should I proceed to make this change?

Reply:
Just add a new commit to your branch, it should appear here automatically as long as the merge request is open.

"I wish we had a place for various cool examples." Well, we should create a new "Recipes" or "Tips and tricks" file inside the Tutorials section, and compile these ideas.

Reply:
Thanks @jqueraltmo. I added a new commit with the change. I like your idea of a Tips section. But I also think there is still a place for some tips embedded with the Reference topic, particularly if it is very specific - a tip on how to use a function. 

I will resolve this conversation.